src/common/config_parser.cpp

```diff
@@ -194,6 +194,14 @@ void ConfigParser::addOptionsModel(cli::CLIWrapper& cli) {
   cli.add<int>("--dim-emb",
       "Size of embedding vector",
       512);
+  cli.add<int>("--factors-dim-emb",
+      "Embedding dimension of the factors. Only used if concat is selected as factors combining form");
+  cli.add<std::string>("--factors-combine", 
+    "How to combine the factors and lemma embeddings. Options available: sum, concat",
+    "sum");   
+  cli.add<std::string>("--factor-predictor",
+      "Method to use when predicting target factors. Options: soft-transformer-layer, hard-transformer-layer, lemma-dependent-bias, re-embedding",
+      "soft-transformer-layer");
```

Comment:
For the target-factors, since different decoding options were already implemented, a more user-friendly config variable was introduced to make more clear which are the available options